src/Data/Argonaut/Core.purs

```diff
@@ -185,8 +185,12 @@ foreign import fromBoolean :: Boolean -> Json
 -- | Construct `Json` from a `Number` value
 foreign import fromNumber :: Number -> Json
 
--- | Construct `Json` from a `String` value. If you would like to parse a string
--- | of JSON into valid `Json`, see `jsonParser`.
+-- | Construct `Json` from a `String` value.
+-- | Note that this function only produces a `Json` object containing a single piece of `String`
+-- | data (similar to `fromBoolean`, `fromNumber`, etc.). These "fromX" functions are often
+-- | used to build-up JSON objects manually, as opposed to using `encodeJson`.
+-- | This function does NOT convert the `String` encoding of a JSON object to `Json` - For that
```

Comment:
```suggestion
-- | This function does NOT parse a string of JSON data into the equivalent `Json` -- for that
```

Reply:
I'm a little wary of using the term 'object' here as it may lead a reader to believe this only applies to literal objects

Reply:
Forgot to mention this, but I'd like to take out this line:

>  These "fromX" functions are often used to build-up JSON objects manually, as opposed to using `encodeJson`.

as there are more libraries for working with JSON than argonaut-codecs (even though it's the one bundled in to the 'argonaut ecosystem') and _this_ library is more of a common base JSON type and utility functions for any other JSON libraries to work with.